src/Router/src/Target/Namespaced.php

```diff
@@ -30,16 +30,12 @@ final class Namespaced extends AbstractTarget
     private $inflector;
 
     /**
-     * @param string $namespace
-     * @param string $postfix
-     * @param int    $options
      * @param string $defaultAction
      */
     public function __construct(
         string $namespace,
         string $postfix = 'Controller',
-        int $options = 0,
-        string $defaultAction = 'index'
```

Comment:
@roxblnfk What do you think about this argument?

Reply:
`__construct` is a special method which doesn't need to follow parent count parameter, and the `defaultAction` parameter is never used in the class method stmts, so I think removal is safe.

Reply:
Looks OK